### PR TITLE
modified standard version to 4.2.1 and debhelper to 11 

### LINKED ISSUE #81: Standard version and Debhelper out of date according to Debian policy
I'm submitting a ...
  

- [ ] bug report

 - [x]  feature request

**Current behavior:**
Currently there is outdated standard version and debhelper version .

**Expected behavior:**
there should be updated standard version and debhelper version according to debian policy.

### PR DESCRIPTION
**Pull request by Manas-kashyap**
#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia) and my PR follows them.
- [x] My branch is up-to-date with the Upstream `master` branch.
- [x] I have added necessary documentation (if appropriate)
- [x] Added breif description

#### Changes proposed in this pull request:
Changes in debian/control , and debian/compat for standard version to 4.2.1 and debhelper 11 

Thank you!

### CODE REVIEW

File: meilix-artwork/debian/control [hunk 1]
```diff
@@ -2,8 +2,8 @@ Source: meilix-artwork
 Section: x11
 Priority: optional
 Maintainer: Mario Behling <mb@mariobehling.de>
-Build-Depends: debhelper (>= 8.0.0), python(>=2.6)
-Standards-Version: 4.1.3
+Build-Depends: debhelper (>= 11), python(>=2.6)
```
Comment (pranav1698) [inline meilix-artwork/debian/control]: In the travis build,
`dpkg-checkbuilddeps: Unmet build dependencies: debhelper (>= 11)
dpkg-buildpackage: warning: build dependencies/conflicts unsatisfied; aborting
dpkg-buildpackage: warning: (Use -d flag to override.)
debuild: fatal error at line 1364:
dpkg-buildpackage -rfakeroot -D -us -uc failed`
Please check for travis in local machine
Weirdly the build is passing

Comment (Manas-kashyap) [reply meilix-artwork/debian/control]: well , what i am thinking it is for signing thing error , as you tested and its passing locally on your machine too

Comment (pranav1698) [reply meilix-artwork/debian/control]: No, You can see it says `unmet build dependencies` travis is not  same as the local machine
Things now to do will be deploying your application to the travis enviroment and see if you get an error there
If you find a solution then you can update the pr

File: meilix-artwork/debian/changelog [hunk 2]
```diff
@@ -1,3 +1,11 @@
+meilix-artwork (1.0-1) unstable; urgency=low
```
Comment (pranav1698) [inline meilix-artwork/debian/changelog]: We have changed the version to 1.0-2

File: meilix-artwork/debian/control (hunk 1 above)
Comment (pranav1698) [inline meilix-artwork/debian/control]: Change the debhelper version to 9

File: meilix-artwork/debian/compat [hunk 3]
```diff
@@ -1 +1 @@
-10
+11
```
Comment (pranav1698) [inline meilix-artwork/debian/compat]: Do the suggested changes and change this to 9

**Comment by abishekvashok** [comment]
Thanks

**Pull request merged**